src/runtime/x86_cpu_features.cpp

```diff
@@ -7,7 +7,7 @@ namespace Internal {
 
 extern "C" void x86_cpuid_halide(int32_t *);
 
-static __attribute__((always_inline)) void cpuid(int32_t fn_id, int32_t *info) {
+static ALWAYS_INLINE void cpuid(int32_t fn_id, int32_t *info) {
```

Comment:
This is now "static inline" linkage. I'm not sure what that means. Probably safe to remove the "static".

Reply:
Fixed